share-through/share-through-htb-system-tests.js

```diff
@@ -57,6 +57,13 @@ function validateBidRequestWithPrivacy(request) {
   expect(queryObj.consent_string).toEqual("TEST_GDPR_CONSENT_STRING");
 }
 
+function validateBidRequestWithUspapi(request) {
```

Comment:
```suggestion
function validateBidRequestWithUspApi(request) {
```
Just suggesting fixing the camel case cause I happen to read that "US Papi"

Reply:
should be `USPApi` then since it stands for `U.S. Privacy Api`

Reply:
we read it exactly like you @maphe lol 😆. this is how IX named the test function (and the related CCPA modules, etc.). i actually think this might be iab naming 🙃

Reply:
ah they're like imposed names? I'll deal with it then...